ci/prow/check-commit-content

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+#FIXME: check if there is a prow github token that can be use to increase the api rate limit from 60 req/h to 5000 req/h
+
+pr=$(curl -H 'Accept: application/vnd.github+json' \
```

Comment:
this can probably be done via git:
git ls 
git ls-remote

Reply:
This json object is needed in order to extract the PR `body` and PR `title`.
PRs are a Github thing and not related to Git.

How would you get the PR's `body` and `title` using Git commands?

Reply:
you are right, it won't get all the details, scratch my comment

Reply:
what bout using gh CLI?

Reply:
I wanted to avoid it because it is requiring authentication...

After [this](https://github.com/kubernetes-sigs/kernel-module-management/pull/28#issuecomment-1234328139) I may transition to `gh` client for consistency and simplicity as prow supports `GITHUB_TOKEN`.

Reply:
Moved to `gh` client.